episodes/operations.md

```diff
@@ -51,6 +51,16 @@ in the Incubator is ready to be incorporated into one of The Carpentries _Lesson
 the collection of lessons and curricula that can be taught in an official workshop for
 Software, Library, or Data Carpentry.
 
+|                                       | Incubator          | Lab                |
+| ------------------------------------- | ------------------ | ------------------ |
```

Comment:
```suggestion
|                                       | Incubator | Lab |
|:--------------------------------------|:---:|:---:|
```
Control relative width of columns, and alignment of cell values. (See the section of the Workbench documentation on tables](https://carpentries.github.io/sandpaper-docs/episodes.html#tables) and linked pandoc documentation therein for details.)